src/Selenium2Library/keywords/_element.py

```diff
@@ -483,10 +483,13 @@ def press_key(self, locator, key):
         lead by '\\\\'.
 
         Examples:
-        | Press Key | text_field   | q |
-        | Press Key | login_button | \\\\13 | # ASCII code for enter key |
+        | Press Key | text_field   | q                |                                               |
+        | Press Key | login_button | \\\\13           | # ASCII code for enter key                    |
+        | Press Key | nav_console  | \\\\\\\\ARROW_UP | # selenium.webdriver.common.keys ARROW_UP KEY |
         """
-        if key.startswith('\\') and len(key) > 1:
+        if key.startswith('\\\\') and len(key) > 1:
```

Comment:
Is it possible we could look for something other than four slashes?

If there's a double slash is it always numbers?